dragonfly/session/session.go

```diff
@@ -161,6 +161,23 @@ func (s *Session) SendJukeBoxPopup(message string) {
 	})
 }
 
+// SendScoreboard ...
+func (s *Session) SendScoreboard(displayName string, objName string){
+	s.writePacket(&packet.SetDisplayObjective{
+		DisplaySlot:   "sidebar",
+		ObjectiveName: objName,
+		DisplayName:   displayName,
+		CriteriaName:  "dummy",
+		SortOrder:     0,
+	})
+}
+// RemoveScoreboard ...
```

Comment:
Please add a newline before this comment.

Reply:
fixed